tests/PHPUnit/Integration/DataAccess/ModelTest.php

```diff
@@ -687,6 +687,23 @@ public function testDeleteInvalidationsForDeletedSites()
         ], $invalidations);
     }
 
+    public function testGetInvalidationsInProgress()
+    {
+        $this->insertInvalidations([
+            ['idsite' => 1, 'date1' => '2014-02-01', 'date2' => '2014-02-28', 'period' => 2, 'name' => 'done'],
+            ['idsite' => 1, 'date1' => '2014-02-01', 'date2' => '2014-02-01', 'period' => 1, 'name' => 'doneb321434abb5a139c17dadf08c9d2e315', 'ts_started' => '2014-02-03 23:13:00', 'status' => 1],
+            ['idsite' => 1, 'date1' => '2014-02-03', 'date2' => '2014-02-03', 'period' => 1, 'name' => 'done', 'ts_started' => '2014-02-03 23:00:00', 'status' => 1],
+            ['idsite' => 1, 'date1' => '2014-02-01', 'date2' => '2014-02-01', 'period' => 1, 'name' => 'done'],
+        ]);
+
+        $expectedInvalidations = [
+            ['idsite' => 1, 'date1' => '2014-02-03', 'date2' => '2014-02-03', 'period' => 1, 'name' => 'done', 'ts_started' => '2014-02-03 23:00:00'],
```

Comment:
```suggestion
            ['idsite' => 1, 'date1' => '2014-02-03', 'date2' => '2014-02-03', 'period' => Day::PERIOD_ID, 'name' => 'done', 'ts_started' => '2014-02-03 23:00:00'],
```

Reply:
Just a minor suggestion here. I understand to update the period IDs across all the tests (or even just the ones touched here) is a bigger job, so it's ok to disregard it.